Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:6
+
+RUN yum update -y && yum install -y wget \
+  && wget http://ftp.rediris.es/mirror/fedora-epel/6/i386/epel-release-6-8.noarch.rpm && yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm \
```

Comment:
You should chose a more "neutral" mirror. Not everybody downloading/using this is located in Spain ;)